aep/0131.yaml

```diff
@@ -0,0 +1,46 @@
+aliases:
+  GetOperation:
+    description: A Get operation is a get on path that ends in a path parameter
+    targets:
+      - formats: ['oas2', 'oas3']
+        given:
+          - $.paths[?(@path.match(/\}']$/))].get
+rules:
+  aep-131-http-body:
+    description: A get operation must not accept a request body.
+    severity: error
+    formats: ['oas3']
+    given:
+      - '#GetOperation.requestBody'
+    then:
+      function: falsy
+
+  aep-131-operation-id:
+    description: Verifies that the operation ID is of the form getresource
+    message: The operation ID does not conform to AEP-131
+    severity: error
+    formats: ['oas2', 'oas3']
+    given: '#GetOperation'
+    then:
+      function: schema
+      functionOptions:
+        schema:
+          type: object
+          properties:
+            operationId:
+              type: string
+              pattern: '^(:|[Gg][Ee][Tt][A-Z]).*$'
+          required: ['operationId']
+
+  aep-131-response-schema:
```

Comment:
will this include custom methods? if so, I don't think we can require that the response of a GET custom method resource-method is the resource itself. e.g. if the Get request is some sort of calculation, perhaps a fetch that reads multiple subresources and returns values that they filter on.

Reply:
How should we distinguish a standard GET from a custom GET? Do we use the ":" in the operationId? Or do we invent a spec extension that must be present in custom get method descriptions?

Reply:
I think as written this is actually okay, but I believe the `GetOperation` selector won't actually match on a custom method like `books/{book}:archive`. So I think it's fine to resolve this coment.

But to your question: yes, we would just only match on the resource-oriented matcher. Or exclude custom methods by checking if there is `:` in the path.